src/lib/components/DrawerCourse/index.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BaseStateElement} from '../BaseStateElement';
+import {html} from 'lit-element';
+import {markAsCompleted} from '../../actions';
```

Comment:
I think we may be able to simplify this since we don't currently have a use case for sharing the course state with the rest of the site.

So I'd suggest removing the changes to actions.js and store.js and making this element extend from BaseElemetn instead of BaseStateElement. In this element's constructor you can run the steps to read the cookies and set this.courseProgess. Then in connectedCallback you can have it mutate its children.

I'm leaning toward keeping things as simple as possible until we have an actual use case for shared state.